CONTRIBUTING.md

```diff
@@ -2,15 +2,15 @@
 Welcome! Thanks for checking out the new Expensify app and for taking the time to contribute!
 
 ## Getting Started
-If you would like to become an Expensify contributor, the first step is to read this document in its entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/App/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, etc). Please read both documents before asking questions, as it may be covered within the documentation.
+If you would like to become an Expensify contributor, the first step is to read this document in its entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/App/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, our app philosophy, etc). Please read both documents before asking questions, as it may be covered within the documentation.
 
 #### Test Accounts
-You can create as many accounts as needed in order to test your changes directly from [the app](https://new.expensify.com/). An initial account can be created when logging in for the first time, and additional accounts can be invited by entering a valid email or phone in the "Find or start a chat" input then tapping the avatar.
+You can create as many accounts as needed in order to test your changes directly from [the app](https://new.expensify.com/). An initial account can be created when logging in for the first time, and you can create additional accounts by starting a chat with a valid email or phone in the "New Chat" or "Group Chat" input, then tapping the avatar.
```

Comment:
```suggestion
You can create as many accounts as needed in order to test your changes directly from [the app](https://new.expensify.com/). An initial account can be created when logging in for the first time, and additional accounts can be created by using the "New Chat" or "Group Chat" option in the Global Create Menu, inputting a valid email or phone number , and tapping the user's avatar.
```

Reply:
idk if `Global Create Menu` is a widely accepted term (referring to the floating green `+` button), but I think this explanation is a bit more succinct imo.

Reply:
+1 -> I kinda like this phrasing even better:

> ... and additional accounts can be created by opening the "New Chat" or "Group Chat" pages via the Global Create menu, inputting a valid email or phone number, and tapping the user's avatar.

Reply:
Awesome, I think my explanation was a bit unclear there, but this one makes a ton of sense!

Reply:
Nice! Team work 👍